### ISSUE TITLE
[Accessibility] The view container title needs to wrap <div> inside of <h3> Header tags.

### ISSUE THREAD

**Issue by Krituhar** [microsoft/vscode#227485]
- VS Code Version: 1.88.0
- OS Version: MacOS Sonoma (v14.6.1)

**Issue**: The contributed extension's view container's title is rendered by the Package.json and it's string is also being updated on runtime in our src code. But the VSCode renders the title of the pane as an `<h3>` element inside of main` <div>` element that has role="button". 

[User Impact]
Assistive technologies may be unable to parse this content.

[Code Reference]
```
<div class="pane-header expanded" tabindex="0" role="button" aria-label="Some string" aria-expanded="true" draggable="true" (...)>
(...)
<h3 class="title" custom-hover="true">Some string</h3>
(...)
</div>
```

[Recommendation]
Ensure ARIA roles, states, and properties are valid. 

In this case, rearrange the code so that the role="button" element is a descendant of the heading element instead.

[Compliant Code Example]
```
<h4>
<div class="pane-header expanded" tabindex="0" role="button" aria-label="Some string" aria-expanded="true" aria-controls="panel1" draggable="true" (...)>
(...)
<div class="title" custom-hover="true">Some string</div>
(...)
</div>
<h4>
```

**Comment by meganrogge** [comment]
No users have reported this, so closing this
